heartbeat/monitors/wrappers/monitors.go

```diff
@@ -0,0 +1,95 @@
+// Licensed to Elasticsearch B.V. under one or more contributor
+// license agreements. See the NOTICE file distributed with
+// this work for additional information regarding copyright
+// ownership. Elasticsearch B.V. licenses this file to you under
+// the Apache License, Version 2.0 (the "License"); you may
+// not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+package wrappers
+
+import (
+	"time"
+
+	"github.com/elastic/beats/heartbeat/eventext"
+	"github.com/elastic/beats/heartbeat/look"
+	"github.com/elastic/beats/heartbeat/monitors/jobs"
+	"github.com/elastic/beats/libbeat/beat"
+	"github.com/elastic/beats/libbeat/common"
+)
+
+// WrapCommon applies the common wrappers that all monitor jobs get.
+func WrapCommon(js []jobs.Job, id string, name string, typ string) []jobs.Job {
+	return jobs.WrapAll(
+		js,
+		monStatus,
+		monTiming,
+		monMeta(id, name, typ),
+	)
+}
+
+func monMeta(id string, name string, typ string) jobs.JobWrapper {
```

Comment:
mon = monitor? I'm personally not a big fan of too many abbreviations as when reading the code I first always need to figure out what it could mean.

Reply:
@ruflin I'm glad to extend it, but I thought inscrutable short names was the go way 😁 !